client/layout/navigation.js

```diff
@@ -57,24 +55,28 @@ const NavigationPlugin = () => {
 					</Link>
 				</WooNavigationItem>
 			) ) }
-			{ reports.map( ( item ) => {
-				const id = `wc_admin-wc-admin&path=/analytics/${ item.report }`;
-				return (
-					<WooNavigationItem item={ id } key={ id }>
-						<Link
-							className="components-button"
-							href={ getNewPath(
-								persistedQuery,
-								`/analytics/${ item.report }`,
-								{}
-							) }
-							type="wc-admin"
+			{ reports
+				.filter( ( item ) => item.navArgs )
```

Comment:
Perhaps we should put this `.filter()` up above just to stay consistent with how it's done with `pages`?